src/core/public/rendering/_base.scss

```diff
@@ -49,6 +49,12 @@
 
   &.kbnBody--hasHeaderBanner {
     @include kbnAffordForHeader($kbnHeaderOffsetWithBanner);
+
+    // Prevents banners from covering full screen data grids
+    .euiDataGrid--fullScreen {
+      height: auto;
+      top: $kbnHeaderBannerHeight;
+    }
```

Comment:
> I know using EUI classes in Kibana stylesheets is frowned upon

These instances where we **have** to target **all** usages is the exception. So all good there.

Is `height: auto` correct though? Would a calculated vh be better like `calc(100vh - #{$kbnHeaderBannerHeight})`? Total speculation though since I haven't pulled down the PR.

Reply:
Good question. I'm not entirely sure what the reason/history was for applying the current `height: 100%;` on the full screen data grid (as the element is already given `top`, `right`, `bottom`, and `left` positioning styles that would dictate its height), but it needed to be overriden due to the `top` style change when the data grid is in full screen.

The options were to use `calc` as you mention above, or just set it to `auto`, since the height styling didn't appear to be doing anything. I went with `auto` for the sake of simplicity, but if there's something I'm overlooking, let me know. Thanks!

Reply:
Ah because of the `height: 100%;` which is being calculated against the whole window because of `fixed`. Just to **make sure** it spans the full height, I'd go the calculated route.

Reply:
Done!